src/Apps/Auction/Components/AuctionDetails/__tests__/AuctionDetails.jest.tsx

```diff
@@ -85,4 +85,36 @@ describe("AuctionDetails", () => {
     const wrapper = getWrapper()
     expect(wrapper.find("AuctionInfoSidebarFragmentContainer").length).toBe(1)
   })
+
+  describe("cascading end time interval message", () => {
+    it("singular form", () => {
+      const wrapper = getWrapper({
+        Sale: () => ({
+          isClosed: false,
+          cascadingEndTimeIntervalMinutes: 5,
+          artworksConnection: {
+            totalCount: 1,
+          },
+        }),
+      })
+
+      const message = "Lot close at 5-minute intervals"
```

Comment:
**thought (blocking):** This doesn't seem right to me. If there is a single lot, there are no intervals between lots, so this message doesn't really make sense. Maybe it makes more sense to suppress this message altogether in the single-lot case?

Looking at the screenshot in the description, I'd say this problem applies to all three messages — maybe we can clarify what the correct copy should be, or if there should be any message at all in the single-lot case? 

"X days until lot closes" (the one in blue) might be the only one that really works?